### ISSUE TITLE
Error predict

### ISSUE THREAD

**Issue by flomato** [george-hall-ucl/dawnn_vignette#1]
Hello,

Trying to run the vignette, but getting the following error: - please advise. Thanks!

> cells <- run_dawnn(cells, label_names = "label", label_1 = "Condition1",
+                    label_2 = "Condition2", reduced_dim = "pca")
WARNING:absl:Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.
WARNING:absl:Error in loading the saved optimizer state. As a result, your model is starting with a freshly initialized optimizer.
Finding neighbors.
Computing nearest neighbors
Only one graph name supplied, storing nearest-neighbor graph only
Generating neighbor labels.
Creating adjacency matrix.
Generating scores.
Error in UseMethod("predict") : 
  no applicable method for 'predict' applied to an object of class "c('keras.src.models.sequential.Sequential', 'keras.models.sequential.Sequential', 'keras.src.models.model.Model', 'keras.models.model.Model', 'keras.src.backend.tensorflow.trainer.TensorFlowTrainer', 'keras.backend.tensorflow.trainer.TensorFlowTrainer', 'keras.src.trainers.trainer.Trainer', 'keras.trainers.trainer.Trainer', 'keras.src.layers.layer.Layer', 'keras.layers.layer.Layer', 'keras.src.backend.tensorflow.layer.TFLayer', 'keras.backend.tensorflow.layer.TFLayer', 'keras.src.backend.tensorflow.trackable.KerasAutoTrackable', 'keras.backend.tensorflow.trackable.KerasAutoTrackable', 'tensorflow.python.trackable.autotrackable.AutoTrackable', 'tensorflow.python.trackable.base.Trackable', 'keras.src.ops.operation.Operation', 'keras.ops.operation.Operation', 'keras.src.saving.keras_saveable.KerasSaveable', 'keras.saving.keras_saveable.KerasSaveable', 'python.builtin.object')"

**Comment by george-hall-ucl** [comment]
Closed as this is a duplicate of https://github.com/george-hall-ucl/dawnn/issues/4.
